java/mykola-melnyk/src/com/zoolatech/lecture2/tasks/_1/CalculatorTest.java

```diff
@@ -0,0 +1,9 @@
+package com.zoolatech.lecture2.tasks._1;
+
+public class CalculatorTest {
```

Comment:
thanks for adding the test

Reply:
Thank you, for your time spent on checking my homework ;)